cypress/e2e/expand-minimize-widget.cy.ts

```diff
@@ -0,0 +1,43 @@
+describe('Widgets autosize and minimize properly', () => {
+  beforeEach(() => {
+    cy.loadLandingPage();
+  });
+
+  afterEach(() => {
+    cy.resetToDefaultLayout();
+  });
```

Comment:
do we need to reset after each test? It seems to be already part of the `loadLandingPage` command